### PR TITLE
National Computing Infrastructure (NCI), Site specific configuration

### PR DESCRIPTION
**Pull request by rbeucher**
## Description

I have added the configuration for our system down here in Australia (VP added: NCI compute cluster data paths in config-developer and pointer to users in config-user)

***

## [Before you get started](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#getting-started)

- [ ] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [X] [🧪][2] The new functionality is [relevant and scientifically sound](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#scientific-relevance)
- [X] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [X] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)/contributing.html#documentation) is available
- [x] [🛠][1] [Unit tests](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#tests) have been added
- [X] [🛠][1] Changes are [backward compatible](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility)
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

File: esmvalcore/config-developer.yml [hunk 1]
```diff
@@ -109,7 +112,7 @@ obs4MIPs:
     ESGF: '{project}/{dataset}/{version}'
     RCAST: '/'
     IPSL: '{realm}/{short_name}/{freq}/{grid}/{institute}/{dataset}/{latest_version}'
-  input_file:
+  input_file:
```
Comment (rbeucher) [inline esmvalcore/config-developer.yml:115]:
```suggestion
  input_file:
```

File: esmvalcore/config-user.yml [hunk 2]
```diff
@@ -247,3 +247,23 @@ drs:
...
+#  ana4mips: /g/data/ct11/access-nri/replicas/esmvaltool/obsdata-v2
+#  native6: /g/data/xp65/public/apps/esmvaltool/native6
+#
+#drs:
+#  CMIP6: NCI
+#  CMIP5: NCI
+#  CMIP3: NCI
+#  CORDEX: ESGF
+#  obs4MIPs: default
+#  ana4mips: default
```
Comment (rbeucher) [inline esmvalcore/config-user.yml:269]:
```suggestion
#  ana4mips: default

```

**Review by valeriupredoi — CHANGES_REQUESTED** [review]
looks good and it's very welcome too, thanks @rbeucher! The only bit missing is adding a test for data finding on the new machine - that should be included here, but if you are not at ease plopping that test here, then I'd suggest letting this be merged, but open an issue and assign one of us tech cores to add it :beer:

**Comment by rbeucher** [comment]
Thanks @valeriupredoi. I have added some tests. Anything else you think is missing?

**Comment by rbeucher** [comment]
OK...it's failing of course. I'm looking into it...

**Review by valeriupredoi — APPROVED** [review]
brilliant! Very many thanks @rbeucher :beer:

**Pull request merged**